input/postgres/replication.go

```diff
@@ -87,19 +85,7 @@ func GetReplication(ctx context.Context, logger *util.Logger, db *sql.DB, postgr
 		return repl, nil
 	}
 
-	if StatsHelperExists(ctx, db, "get_stat_replication") {
-		logger.PrintVerbose("Found pganalyze.get_stat_replication() stats helper")
-		sourceTable = "pganalyze.get_stat_replication()"
-	} else {
-		if systemType != "heroku" && !connectedAsSuperUser(ctx, db, systemType) && !connectedAsMonitoringRole(ctx, db) {
-			logger.PrintInfo("Warning: You are not connecting as superuser. Please setup" +
-				" the monitoring helper functions (https://github.com/pganalyze/collector#setting-up-a-restricted-monitoring-user)" +
-				" or connect as superuser, to get replication statistics.")
-		}
-		sourceTable = "pg_stat_replication"
-	}
-
-	rows, err := db.QueryContext(ctx, QueryMarkerSQL+fmt.Sprintf(replicationStandbySQL, sourceTable))
+	rows, err := db.QueryContext(ctx, QueryMarkerSQL+replicationStandbySQL)
```

Comment:
Hmm, I realized we don't actually want to do this, because Aiven still does not grant pg_monitor as far as I know. I'll revert this. I guess we should add a get_stat_replication helper page in the official docs.